src/views/__tests__/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/no-commonjs
```

Comment:
are these needed in every subdirectory? will eslint not fine the root config by default?

Reply:
Maybe better question is why `.js`. json or yaml would work without any such comments.

Reply:
Also the config should set it by default: https://github.com/satya164/eslint-config-satya164/blob/master/index.js#L162